luigi/rpc.py

```diff
@@ -110,9 +110,9 @@ def ping(self, worker):
         # just one attemtps, keep-alive thread will keep trying anyway
         self._request('/api/ping', {'worker': worker}, attempts=1)
 
-    def add_task(self, worker, task_id, status=PENDING, runnable=False,
-                 deps=None, new_deps=None, expl=None, resources={}, priority=0,
-                 family='', params={}):
+    def add_task(self, worker, task_id, status=PENDING, runnable=True,
+                 deps=None, new_deps=None, expl=None, resources=None, priority=0,
+                 family='', params=None):
```

Comment:
I changed the default value here to respect the same values as CentralScheduler. I'm not sure if the two schedulers didn't have the same signature.